pkg/presidio/presidio.go

```diff
@@ -19,7 +19,7 @@ type ServicesAPI interface {
 	SetupDatasinkService()
 	SetupRecognizerStoreService()
 	SetupCache() cache.Cache
-	AnalyzeItem(ctx context.Context, text string, template *types.AnalyzeTemplate) ([]*types.AnalyzeResult, error)
+	AnalyzeItem(ctx context.Context, text string, template *types.AnalyzeTemplate) (*types.AnalyzeResponse, error)
```

Comment:
This has to go with the presidio-genproto change. Correct? if we want to do it in two phases, this should be reverted.

Reply:
To my understanding of Elad's code, this is a response of the API, and therefore, is not changing the genproto

Reply:
I'm not sure. Check out https://github.com/microsoft/presidio-genproto/pull/22/files

Reply:
correct